src/NServiceBus.Core.Analyzer.Tests.Common/FeatureDefaultsEnableFeatureAnalyzerTests.cs

```diff
@@ -0,0 +1,158 @@
+#pragma warning disable NUnit1034 // Base TestFixtures should be abstract
+
+namespace NServiceBus.Core.Analyzer.Tests;
+
+using System.Threading.Tasks;
+using Helpers;
+using NUnit.Framework;
+
+[TestFixture]
+public class FeatureDefaultsEnableFeatureAnalyzerTests : AnalyzerTestFixture<FeatureDefaultsEnableFeatureAnalyzer>
+{
+    [Test]
+    public Task DiagnosticIsReportedForExpressionLambda()
+    {
+        var source =
```

Comment:
We should use `"""` delimited code blocks for all new tests. I'm sure this was the agent following the overwhelming pattern in the rest of the tests that we built before that feature existed.